packages/aws-rfdk/lib/lambdas/nodejs/x509-certificate/acm-handlers.ts

```diff
@@ -94,11 +94,14 @@ export class AcmCertificateImporter extends DynamoBackedCustomResource {
         inUseByResources = cert!.InUseBy || [];
 
         if (inUseByResources.length) {
-          // Exponential backoff with jitter based on 200ms base
+          // Exponential backoff with jitter based on 1s base, to a max of 30s,
           // component of backoff fixed to ensure minimum total wait time on
           // slow targets.
           const base = Math.pow(2, attempt);
-          await defaultSleep(Math.random() * base * 50 + base * 150);
+          const backoffDelay = Math.floor(Math.random() * base * 200 + base * 1000 - 100);
```

Comment:
I've increased the delay from a base of 200ms. I figure we're waiting for the ACM service here to update the certificate; possibly also waiting for the ALB to signal that it's no longer using it as well.

So, rather than waiting (max): 0.2s, 0.35s, 0.65s, 1.250s, 2.45s, 4.85s, 9.65s, 19.25s (which won't finish due to 30s lambda timeout)

It might make more sense to wait longer, but cap out the delay (max) 1.1s, 2.1s, 4.1s, 8.1s, 16.1s, 30s, 30s, ....